src/ServiceControl.Persistence/IPersistenceLifecycle.cs

```diff
@@ -3,9 +3,8 @@
     using System.Threading;
     using System.Threading.Tasks;
 
-    public interface IPersistenceLifecycle // TODO: We don't need this, implementations should just implement IHostedService
+    public interface IPersistenceLifecycle
```

Comment:
Since all the implementations implement IDisposable, shouldn't this as well?

Reply:
Implementations currently use IDisposable but could also use IAsyncDisposable. The interface should not demand one over the other.